src/AppInstallerCLIE2ETests/Helpers/WinGetSettingsHelper.cs

```diff
@@ -196,5 +202,24 @@ public static void InitializeAllFeatures(bool status)
             ConfigureFeature("windowsFeature", status);
             ConfigureFeature("download", status);
         }
+
+        private static JObject GetJsonSettingsObject(string objectName, string propertyName)

```

Comment:
What does this function do?
It seems like it parses the settings file, then adds an object with a single empty property?
That is weird to me, and the method name doesn't help.

Reply:
What about something like

```C#
private static JObject GetJsonSettingsObjectAndCreateEmptyProperty(string[] propertyPath...)
{
    JObject settingsJson = JObject.Parse(File.ReadAllText(TestSetup.Parameters.SettingsJsonFilePath));
    var currentNode = settings.Json;
    foreach (var key in propertyPath)
    {
        if (!currentNode.ContainsKey(key))
        {
            currentNode[key] = new JObject();
        }
        currentNode = currentNode[key];
    }

    return settingsJson;
}
```

(Not sure if this even compiles...)

Reply:
I changed the helper method to be a little more generic so that it only returns the json settings object that you specify and doesn't do anything with creating a single empty property (which I also agree is kind of weird).